subprojects/gradle/performance/src/main/kotlin/com/avito/performance/PerformanceExtension.kt

```diff
@@ -25,4 +26,21 @@ open class PerformanceExtension @Inject constructor(objects: ObjectFactory) {
      * Slack hook url to send urgent info
      */
     val slackHookUrl = objects.property<String>()
+
+    /**
+     * Toggle for different strategies of target branch results fetching
+     */
+    @Suppress("UnstableApiUsage")
+    val targetBranchResultSource: Property<TargetBranchResultSource> =
+        objects.property<TargetBranchResultSource>()
+            .convention(TargetBranchResultSource.RUN_IN_PROCESS)
+
+    /**
+     * FETCH_FROM_OTHER_BUILD strategy needs to know about build config id
+     */
+    val targetBuildConfigId = objects.property<String>()
```

Comment:
Can you move this parameter into FETCH_FROM_OTHER_BUILD strategy dsl?

```
build.gradle:

performance {
     targetBranchResultSource = runInProcess()
}

performance {
     targetBranchResultSource = fetchFromOtherBuild(id = "build id")
}
```
Flat parameters have more inconsistent states. It's harder to understand and validate

Reply:
will do; was afraid of groovy/kotlin compatibility